packages/config/src/index.ts

```diff
@@ -55,39 +53,28 @@ export default class ConfigManager implements IConfigManager {
     this.sdk
```

Comment:
Either change this to an "await" only approach, or revert it. Mixing awaits and then/catch Promises is very ugly.